src/System.Net.Requests/tests/RequestStreamTest.cs

```diff
@@ -11,11 +11,18 @@
 
 namespace System.Net.Tests
 {
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Tests hang on .NET Framework")]
     public class RequestStreamTest
     {
         readonly byte[] buffer = new byte[1];
 
+        static RequestStreamTest()
+        {
+            // On .NET Framework, the default limit for connections/server is very low (2).
+            // On .NET Core, the default limit is higher. Since these tests run in parallel,
+            // the limit needs to be increased to avoid timeouts when running the tests.
+            ServicePointManager.DefaultConnectionLimit = int.MaxValue;
```

Comment:
So basically the tests are creating a leak, and rather than fixing the leak we're increasing the size of the bucket?  It'd be better to just fix the leak, e.g. if the problem is that GetRequestStream is leaving open a connection because we haven't actually asked for the response, we should ask for the response and dispose of it.